entity-framework/core/modeling/spatial.md

```diff
@@ -23,22 +23,19 @@ Microsoft.EntityFrameworkCore.SqlServer | [Microsoft.EntityFrameworkCore.SqlServ
 Microsoft.EntityFrameworkCore.Sqlite    | [Microsoft.EntityFrameworkCore.Sqlite.NetTopologySuite](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.Sqlite.NetTopologySuite)
```

Comment:
Can remove the  'EF Core 2.2' note

Should we add a column for the first version where spatial is supported for each provider?

Reply:
I want to keep the 2.2 note until 2.1 goes out of support. All providers work on EF Core 3.1.